src/elements/fields/QRScanner/index.tsx

```diff
@@ -53,7 +55,21 @@ function QRScanner({
         if (editMode || !decodedText) return;
         if (decodedText !== fieldVal) onChange(decodedText);
       };
-      scanner.render(onSuccess);
+
+      const onError = (error: any) => {
```

Comment:
this onError can also be pulled out of the component, maybe renamed to onQRError